### ISSUE TITLE
In AWS EKS the list of pods is visible from `kubectl` not from Lens.

### ISSUE THREAD

**Issue by danidemi** [lensapp/lens#2609]
**Describe the bug**
I'm using EKS on AWS.
If I use `kubectl` I can get the list of running pods, their logs and so on.
In lens I cannot access the pods list instead under workload.

**To Reproduce**
I use this procedure to reproduce the bug:

1. From command line I launch  `aws sso login --profile=ic-test`
2. That completes successfully, opens a browser, complete the login, etc...
3. I open Lens and I check the workload section, nothing there.
4. But when I issue a kubectl get pods from command line, pods are shown

**Expected behavior**
I think I should see the pods in the workload section

**Screenshots**
1) The login to AWS is successful
![image](https://user-images.githubusercontent.com/345826/115876384-8a4e8600-a446-11eb-99d8-35be383a9661.png)

2) Pods are not shown in Workload but I can get them with `kubectl`
![image](https://user-images.githubusercontent.com/345826/115876667-e74a3c00-a446-11eb-89d9-c44b19650aa1.png)

**Environment (please complete the following information):**
- Lens Version: 4.2.2
- OS: Windows10
- Installation method (e.g. snap or AppImage in Linux): Win exe 64 bit, installation for all users

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
PS C:\Program Files\Lens> .\Lens.exe
PS C:\Program Files\Lens>
info: ­ƒôƒ Setting Lens as protocol client for lens://
info: ­ƒôƒ succeeded Ô£à
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: LS0tLS1CRUdJT...JTm9ZRElrUjkyWWoybGpKNm12eUNheWJ2VT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
    server: https://925334FBF88268531E9A9EC20021795F.sk1.eu-west-1.eks.amazonaws.com
  name: SVTS-K8s-K8s-SVTS-Cluster
- cluster:
    certificate-authority-data: LS0tLS1CRUdJ...NBVEUtLS0tLQo=
    server: https://0979ECE0F56E8A969DFD36153BDF74FD.yl4.eu-west-1.eks.amazonaws.com
  name: arn:aws:eks:eu-west-1:956251722582:cluster/SVTS-Factory-EKS-Cluster
- cluster:
    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJ...lGSUNBVEUtLS0tLQo=
    server: https://kubernetes.docker.internal:6443
  name: docker-desktop
- cluster:
    certificate-authority-data: LS0tLS1CRUdJ...0VSVElGSUNBVEUtLS0tLQo=
    server: https://34.76.197.49
  name: gke_zcs-smarty-cloud_europe-west1_gke-smarty-cloud
- cluster:
    certificate-authority: C:\Users\esche\.minikube\ca.crt
    server: https://172.17.244.185:8443
  name: minikube
contexts:
- context:
    cluster: arn:aws:eks:eu-west-1:956251722582:cluster/SVTS-Factory-EKS-Cluster
    namespace: id-smsa-svts-platform-namespace
    user: arn:aws:eks:eu-west-1:956251722582:cluster/SVTS-Factory-EKS-Cluster
  name: arn:aws:eks:eu-west-1:956251722582:cluster/SVTS-Factory-EKS-Cluster
- context:
    cluster: docker-desktop
    user: docker-desktop
  name: docker-desktop
- context:
    cluster: docker-desktop
    user: docker-desktop
  name: docker-for-desktop
- context:
    cluster: gke_zcs-smarty-cloud_europe-west1_gke-smarty-cloud
    user: gke_zcs-smarty-cloud_europe-west1_gke-smarty-cloud
  name: gke_zcs-smarty-cloud_europe-west1_gke-smarty-cloud
- context:
    cluster: SVTS-K8s-K8s-SVTS-Cluster
    namespace: identity-smsa
    user: devops-infocert-test
  name: infocert-test
- context:
    cluster: minikube
    user: minikube
  name: minikube
current-context: arn:aws:eks:eu-west-1:956251722582:cluster/SVTS-Factory-EKS-Cluster
kind: Config
preferences: {}
users:
- name: arn:aws:eks:eu-west-1:956251722582:cluster/SVTS-Factory-EKS-Cluster
  user:
    exec:
      apiVersion: client.authentication.k8s.io/v1alpha1
      args:
      - --region
      - eu-west-1
      - eks
      - get-token
      - --cluster-name
      - SVTS-Factory-EKS-Cluster
      command: aws
      env:
      - name: AWS_PROFILE
        value: ic-test
- name: devops-infocert-test
  user:
    exec:
      apiVersion: client.authentication.k8s.io/v1alpha1
      args:
      - --region
      - eu-west-1
      - eks
      - get-token
      - --cluster-name
      - SVTS-K8s-K8s-SVTS-Cluster
      command: aws
      env:
      - name: AWS_PROFILE
        value: ictst
- name: docker-desktop
  user:
    client-certificate-data: LS0tLS...LS0K
    client-key-data: LS0tL...VktLS0tLQo=
- name: gke_zcs-smarty-cloud_europe-west1_gke-smarty-cloud
  user:
    auth-provider:
      config:
        access-token: ya2...GIRiN0
        cmd-args: config config-helper --format=json
        cmd-path: C:\Users\esche\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin\gcloud.cmd
        expiry: "2021-04-23T14:04:04Z"
        expiry-key: '{.credential.token_expiry}'
        token-key: '{.credential.access_token}'
      name: gcp
- name: minikube
  user:
    client-certificate: C:\Users\esche\.minikube\client.crt
    client-key: C:\Users\esche\.minikube\client.key

```

**Additional context**
Nothing more to add.

**Comment by sudhirski** [comment]
FWIW: I saw the exact same thing on the Lens IDE. 

I just had to manually select a namespace in the "Namespaces" drop down selector, and then the pods showed up fine.

**Comment by Nokel81** [comment]
@danidemi It looks like you have some RBAC rules in place. Lens currently treats the first 10 namespaces as representative of all namespaces in terms of permissions.

There exits #2892 which is targeted for v5 which should help this.

**Comment by pjaak** [comment]
Hi @Nokel81 ,

We are currently having the same issue and just did some testing and indeed our Role and Rolebindings only work on the top ten namespaces but we cant see any resources when we have our Role and Rolebindings for namespaces lower then 10. I see the link you sent above has been closed. Is there still plans to solve this permissions issue for only the top 10 namespaces?

Thanks

**Comment by ashishkarpe** [comment]
Is this issue fixed? I am getting same issue in Lens: 2022.10.311317-latest

**Comment by Nokel81** [comment]
This issue has not been fixed. You can manually specify your "Allowed Namespaces" as a work around.
